spec/models/target_spec.rb

```diff
@@ -9,4 +9,45 @@
     it { should validate_presence_of(:longitude) }
     it { should validate_presence_of(:user) }
   end
+
+  describe 'matches' do
+    let!(:target_1) do
+      create(:target,
+             length: 200, latitude: -34.90644397, longitude: -56.20093115,
+             topic: 'Politics')
+    end
+    let!(:target_2) do
+      create(:target,
+             length: 200, latitude: -34.90733909, longitude: -56.19726226,
+             topic: 'Politics')
+    end
+    let!(:target_3) do
+      create(:target,
+             length: 200, latitude: -34.90876542, longitude: -56.19440534,
+             topic: 'Politics')
+    end
+    let!(:target_4) do
+      create(:target,
+             length: 500, latitude: -34.91056160, longitude: -56.21137661,
+             topic: 'Politics')
+    end
+
+    context 'one target' do
```

Comment:
this could be improved to somethign like
`context 'when the target has only one match`